### ISSUE TITLE
Function not found when using GetRepository

### ISSUE THREAD

**Issue by ktwbc** [Haehnchen/idea-php-symfony2-plugin#805]
`$objDocuments = $this->get('doctrine')->getRepository('AppBundle:Docs')->findAllDisplayable();`

public function findAllDisplayable() exists in DocsRepository, and the code works just fine, but PHPStorm/Symfony plugin flags Method 'findAllDisplayable' not found in \Doctrine\Common\Persistence\ObjectRepository 

Note that Docs Entity does contain 
@ORM\Entity(repositoryClass="AppBundle\Entity\DocsRepository")

**Comment by ktwbc** [comment]
so kind of found the issue. Using getDoctrine() instead of ->get('doctrine') does cause the plugin to resolve correctly. not sure if it's designed that it should work in both scenarios

**Comment by cmodijk** [comment]
I just updated to the latest version and found the same thing but i can't switch to `getDoctrine()` 
because i'm in a CLI command i'm using `$this->getContainer()->get('doctrine')`.

**Comment by cmodijk** [comment]
@ktwbc Yes, But I am wondering what changed in the plugin that this does not work any more :-)

**Comment by Koc** [comment]
Similar to #287
